src/dagmc.cpp

```diff
@@ -72,6 +72,25 @@ DAGUniverse::DAGUniverse(pugi::xml_node node)
     adjust_material_ids_ = get_node_value_bool(node, "auto_mat_ids");
   }
 
+  // get material assignment overloading
+  if (check_for_node(node, "material_overrides")) {
```

Comment:
We'll want to document the updates to the XML specification for `geometry.xml` to include this block in the `DAGMCUniverse` nodes.